elasticsearch/bulk/bulk.go

```diff
@@ -180,11 +185,47 @@ func (b *Bulk) flushMessages() error {
 func (b *Bulk) bulkRequest() error {
 	startedTime := time.Now()
 	reader := bytes.NewReader(b.batch)
-	_, err := b.esClient.Bulk(reader)
+	resp, err := b.esClient.Bulk(reader)
 	b.metric.BulkRequestProcessLatencyMs = time.Since(startedTime).Milliseconds()
 	if err != nil {
 		return err
 	}
+	err = b.handleResp(resp)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func (b *Bulk) handleResp(resp *esapi.Response) error {
+	if resp == nil {
+		return fmt.Errorf("ERROR occurred when bulk request. Error: Response is nil")
```

Comment:
I think we should have maintained the log format for all log types(info, warn, error) for example we don't write "ERROR ..." prefix to error logs so could we re-write error logs like  this one 

https://github.com/Trendyol/go-elasticsearch-connect-couchbase/blob/0f3d86d94424d95127e97db1a6ee1102b47dedf2/elasticsearch/bulk/bulk.go#L111

what do you guys think about it @erayarslan @Abdulsametileri @mhmtszr @emreodabas

Reply:
I updated the log messages.